README.md

```diff
@@ -4,17 +4,49 @@
 
 ![bots shopping to the death](bots.png)
 
-## Running the server
+## Rules of the game
 
-### How to start
+* Bots act as clients. The clients have time until the next tick to register their next move to the server. 
+  If they do it more than once, the previous move is overwritten. 
+  On next tick the server calculates the next game state, which is accessible from another endpoint.
+* Available actions are movement (UP, DOWN, LEFT, RIGHT), picking up an item (PICK) or using one (USE).
+* Faulty responses are penalised by decreasing the bot's health.  Dead
+  bots are removed from the shop by guards.  Faulty actions include
+  running into a wall, trying to pick nonexistent items, etc.
+* The items in the shop have a price and a discount percentage.  The
+  better the discount, the longer it takes to pick up.
+* Every bot has a given amount of money, which is spent on shopping.
+* The higher the value of items collected, the better will be the bot's
+  position on the top list.
+* The "price" of an item is the price *before* discount, so it is *not*
+  the amount of money that will actually be spent when you pick it up.
+* When items have been collected, new ones are added on the map.
+* There's no time limit to exploring the shop.
```

Comment:
There's no time limit to exploring the shop, but your health decreases because shopping is hard.